### PR TITLE
Make sure control-center job runs with no errors

### PR DESCRIPTION
**Pull request by ynurmahomed**
Should be able to execute the run command with no errors on `egpaf_mabil` using this [config.zip](https://github.com/esaude/openmrs-database-migrator/files/3646443/config.zip).

### CODE REVIEW

File: src/main/java/com/openmrs/migrator/Migrator.java [hunk 1]
```diff
@@ -119,10 +119,23 @@ private void executeSetupCommand() throws IOException, SQLException, SettingsExc
     List<String> pdiFiles = new ArrayList<>();
 
     pdiFiles.add(SettingsService.PDI_RESOURCES_DIR + "/jobs/control-center.kjb");
+    pdiFiles.add(SettingsService.PDI_RESOURCES_DIR + "/jobs/preparation.kjb");
```
Comment (nthfloor) [inline src/main/java/com/openmrs/migrator/Migrator.java:122]: @ynurmahomed we are moving away from this approach of hard coding the resources in the Java app - better that we design the app to just read all resources contained under `pdiresources/` folder and treat the pdi files as input files.

Comment (ynurmahomed) [reply src/main/java/com/openmrs/migrator/Migrator.java:122]: I think @cfaife is working on that. For now its the only way to have the job running.

**Comment by ynurmahomed** [comment]
@nthfloor @cfaife can we merge this?

**Comment by nthfloor** [comment]
Has anyone reviewed this yet?

@cfaife @kaweesi 

If not can someone please take a look so we can merge it today

**Pull request merged**